implementations/rust/ockam/ockam_command/src/node/show.rs

```diff
@@ -44,10 +52,14 @@ pub async fn query_status(ctx: ockam::Context, _: (), mut base_route: Route) ->
         ..
     } = api::parse_status(&resp)?;
 
+    let node_cfg = cfg.get_node(&node_name).unwrap();
+    let api_address = SocketAddr::new(IpAddr::from_str("127.0.0.1").unwrap(), node_cfg.port);
+    let (mlog, _) = cfg.log_paths_for_node(&node_name.to_string()).unwrap();
+    let log_path = util::print_path(&mlog);
+
     println!(
-        "Node: {}, Status: {}, Worker count: {}, Pid: {}, Transport count: {}",
-        node_name, status, workers, pid, transports,
+        "\nNode:\n\x20\x20Name: {}\n\x20\x20Status: {}\n\x20\x20API Address: {}\n\x20\x20Pid: {}\n\x20\x20Worker count: {}\n\x20\x20Transport count: {}\n\x20\x20Log Path: {}",
+        node_name, status, api_address, pid, workers,transports , log_path
```

Comment:
You can make this more readable using raw notation like: 

```rust
r#"Node:
  Name: {}
  Status: {}
..."#
```

https://play.rust-lang.org/?version=stable&mode=debug&edition=2015&gist=888c93b50fd35737f80c21e363926b53

Reply:
refactor in [c04f278](https://github.com/build-trust/ockam/pull/3090/commits/c04f278bcc5f2b12c73b860c5e594767886b7e7c)